docs/SOP.md

```diff
@@ -23,6 +25,45 @@ This document contains standard operating procedures (SOPs) for the OBO Foundry
 1. At the next Operations call after that one, the liaison presents the NOR to the OBO Foundry Operations Committee and answers questions. In most cases, the information provided will be sufficient to either grant or refuse the prefix request. In some cases, the committee may choose to postpone its decision to require some clarification and fixes from the submitter.
 The liaison MUST be present at the Operations call in order for the NOR case to be discussed. If the liaison does not participate for 2 consecutive Operations calls, the chair of the second call emails the liaison to request a statement confirming the ability to continue as liaison. If the liaison does not participate in 3 consecutive Operations calls and did not respond to the email above, a new liaison is assigned during that third call.
 
+<a name="ROOM"></a> 
+### Reviewing Ontologies for OBO Membership
+
+The goal of this SOP is to provide a clear set of criteria to be checked for the manual review of an ontology in response to a request to register that new ontology with the OBO Foundry. It is expected that a programmatic review using the Dashboard has already been done and the submitters have addressed any problems found. The purpose of the manual review is to check the ontology for issues that the Dashboard review does not cover. A sample of terms/axioms should be checked. In order for this review to be relatively quick (~ 2 hours), the reviewer is not expected to review all the terms/axioms.
+
+Check the following and provide a brief summary in the tracker issue for the new ontology request. All items of feedback must be provided using GitHub checklist syntax (`- [ ] TODO`) in order to track how far along they are in being addressed. Addressable issues identified as part of the review should be added to the new ontology’s issue tracker. 
+1. Ontology scope. The new ontology must present use cases demonstrating its relevance to the life sciences.  Was the ontology developed using expert input or trusted scientific sources representative of the consensus in its target domain of knowledge? If the ontology was developed for a very specific purpose or community, representation and consensus need not be broad; however, this scope should be clearly stated.
+2. Terms with the new ontology prefix. The new terms provided by the ontology should not already be available in an available OBO Foundry ontology. If there is an overlap of terms, justification should be provided. There must not be a term with the same label available. There should not be another OBO Foundry ontology whose scope covers those terms and if there is there should be some evidence of contacting that OBO Foundry ontology.
```

Comment:
```suggestion
2. Terms with the new ontology prefix. There MUST NOT be a term with the same label available in another OBO Foundry Ontology. There SHOULD NOT be another OBO Foundry ontology whose scope covers any of the new terms. If there is an overlap of terms with another OBO Foundry ontology, justification MUST be provided. An example of an acceptable justification is that the other OBO Foundry ontology was contacted and the request for adding the terms in scope for that ontology where rejected. In this case, evidence of this contact must be provided to the reviewer on request.
```

Reply:
This paragraph is so important, I decided to give it a shot to rewrite it before merging. It does not change the meaning, but I think adds clarity on what exceptions are permissible.

Reply:
@cstoeckert

Reply:
Thank you for reviewing this @matentzn The current text was written in response to the feedback from the Governance Task Team (see the google doc for more on this): 
"Therefore we suggest that this point to be change to 1) use less strong wording (i.e. avoid “must not”), and 2) express that this practice should be avoided, and, when used, should be justified. Also, clear guidance should be given to reviewers to understand how this can be checked (how would one know what term to check?)." 
I think some of your edits go against that. Your suggested justification example could be added though:
"An example of an acceptable justification is that the other OBO Foundry ontology was contacted and the request for adding the terms in scope for that ontology was rejected. In this case, evidence of this contact must be provided to the reviewer on request."

Reply:
there's a lot of MUSTs now, what's the recourse when these aren't followed? Kicked out of OBO?

Reply:
@cthoyt This SOP is for review of ontologies asking to join the OBO Foundry so the recourse is a recommendation against admission. If there are "must"s that you think would be better as "should"s please point them out. Thanks!

Reply:
no, I'm all for the MUSTs! But what happens when an ontology follows the rules to get in but then stops following the rules? For example, I know that my old group is trying to get many ontologies accepted into OBO and it's their _modus operandi_ to abandon things once they can say to the bosses "look we are legitimate because OBO said so"

Reply:
That is definitely an issue but to be covered in a different SOP. :-)

Reply:
I will have this rechecked by the Governance team.

Reply:
There are a lot of "MUST" and "MUST NOT" in the new version, which doesn't give space for exceptions (which we already have a lot in already approved obo ontologies. I know this is a different issue, but one could make the argument). I think the message should be: This must be avoided at all costs, but if it must happen, (good) justification(s) is(are) required.
- "There MUST NOT be a term with the same meaning available in another OBO Foundry Ontology. "
I agree. Basically, we don't want 2 identical terms (ie 2 terms with the same concepts), whether or not the label is the same
- "Where meaning is ambiguous, a clear justification for the interpretation of both terms MUST be given (labels, definitions, axioms)." 
This is a dangerous sentence... It assumes that one would have to argue for the interpretation of a term from an ontology that is not theirs. 

Proposal: 
There MUST NOT be a term with the same meaning available in another OBO Foundry Ontology, ie there must not be a term referring to a concept that already exists in another OBO Foundry Ontology (whether or not the label is identical). There SHOULD NOT be another OBO Foundry ontology whose scope covers any of the new terms. In the event that these conditions cannot be fulfilled, justification(s) must be provided. Such justification(s) include: 
- the demonstration that these terms are actually not the same (this happens when term meaning/concept is ambiguous); or
- the other OBO Foundry ontology was contacted and rejected the request for adding new terms in scope for that ontology. 

(what about terms with same labels? should we add this too? )

Reply:
I like this. I updated my proposal to reflect exactly your suggestion @sabrinatoro. Can you confirm it is ok now?

We don't need to say anything about labels, because labels are only a proxy for meaning anyways.

Reply:
@matentzn It's perfect! Thank you!